who_is_active.sql

```diff
@@ -2049,6 +2123,25 @@ BEGIN;
 									blk.session_id = 0
 									AND @blocker = 0
 								)
+							LEFT JOIN sys.dm_exec_sessions AS es ON

```

Comment:
You're hitting an almost-identical set of tables below on line 3077. I'm not sure I understand why you need to do this work twice? I'd much prefer if you only do it below - there should be no reason to hit dm_exec_sessions/requests more than once, as happens today.

Reply:
Agreed, I removed this from the final version a few commits ago,  not sure why its still lingering, I will upload my changes again without it.

Reply:
I brought the tables up to this level with the latest, but I see we run into issues with used_memory_kb being polled at the incorrect time,  and not having the latest value to match up with the session loops. I think I would keep this at the lower level (line 3077) to consider the polling factor when looping through sessions. Then we would grab the max used_memory_kb right?  Or am I thinking about think incorrectly...either way great learning...